lib/ui/painting.dart

```diff
@@ -1830,6 +1835,49 @@ class Image {
   String toString() => _image.toString();
 }
 
+class ImagePerfData {
```

Comment:
@chinmaygarde what way would you prefer that we track image decode times with images? Would you prefer expandos or should we just add a `debugResourceDecodeTime` field to the `Image` class?